htdocs/core/tpl/objectline_title.tpl.php

```diff
@@ -56,7 +56,24 @@
 }
 
 // Description
-print '<th class="linecoldescription">'.$langs->trans('Description').'</th>';
+print '<th class="linecoldescription">'.$langs->trans('Description');
+// @phan-suppress-next-line PhanUndeclaredConstantOfClass
```

Comment:
Can you remove this comment line ? May hide a bug.

Reply:
The huge problem is that if I remove this line, the CI/phan check fails.
The second point is that I copy this solution from line 91!

????? What should we do ????? ;)

Reply:
Remove the line. If CI/phan check fails after, it means we found a bug to fix. So remove it so we can see what is the error message to have a clue of the bug...